modules/kubernetes-addons/aws-node-termination-handler/locals.tf

```diff
@@ -67,6 +24,16 @@ locals {
     nth-sa-name = local.service_account_name
   })]
 
+  irsa_config = {
+    kubernetes_namespace              = local.namespace
+    kubernetes_service_account        = local.service_account_name
+    create_kubernetes_namespace       = false
+    create_kubernetes_service_account = true
+    iam_role_path                     = "/"
+    tags                              = var.addon_context.tags
+    irsa_iam_policies                 = [aws_iam_policy.aws_node_termination_handler_irsa.arn]
```

Comment:
add `var.irsa_policies` and `concat` policies in the `irsa_config`. See https://github.com/aws-samples/aws-eks-accelerator-for-terraform/blob/cebffac39026e6babad116901a81b5bf258f38b8/modules/kubernetes-addons/keda/locals.tf#L40